src/callbacks_step/analysis.jl

```diff
@@ -634,9 +634,7 @@ pretty_form_utf(quantity) = get_name(quantity)
 pretty_form_ascii(quantity) = get_name(quantity)
 
 # Special analyze for `SemidiscretizationHyperbolicParabolic` such that
-# precomputed gradients are available. For now only implemented for the `enstrophy`
-#!!! warning "Experimental code"
-#    This code is experimental and may be changed or removed in any future release.
```

Comment:
Might be, maybe this also persisted because the twist for `analyze` was at that point only for `enstrophy`, i.e., a relatively small use-case.

Reply:
I decided to remove this now since the viscous forces are not experimental and thus I would not mark an essentially equivalent function with this warning.